cmd/crio/config.go

```diff
@@ -131,6 +131,13 @@ image_volumes = "{{ .ImageVolumes }}"
 insecure_registries = [
 {{ range $opt := .InsecureRegistries }}{{ printf "\t%q,\n" $opt }}{{ end }}]
 
+# registries is used to specify registries to be used when pulling an unqualified
+# image (e.g. fedora:rawhide). "docker.io" is left here as a default registry.
+# Users can remove it and use whatever default registry they need.
+registries = [
+  "docker.io",
```

Comment:
Even if it still defaults to docker.io in containers/image we need to have a way to still have a default for docker.io when pulling unqualified images because that's just what people expect (and what we do in RHEL right now, unless people actually block docker.io).
I'm fine leaving this to distro but bear in mind crio is going to error out if nobody add a registry here.